code/__HELPERS/game.dm

```diff
@@ -271,7 +271,7 @@ var/list/DummyCache = list()
 
 	//Now, check objects to block exit that are on the border
 	for(var/obj/border_obstacle in srcturf)
-		if(border_obstacle.flags & ON_BORDER)
+		if(!(target.flags & ON_BORDER) && (border_obstacle.flags & ON_BORDER))
```

Comment:
I wanted Station Holomaps to be reachable through window panes, I guess I'll give up on that.